lab/pom.xml

```diff
@@ -18,6 +18,7 @@
         <easymock.version>4.3</easymock.version>
         <hibernate.jmx.version>3.5.6-Final</hibernate.jmx.version>
         <jamon.version>2.82</jamon.version>
+        <spring-security.version>5.8.0-RC1</spring-security.version>
```

Comment:
@salmar will need to support in `build.gradle` too.

Reply:
Should be done now